src/FortuneWindow.cpp

```diff
@@ -78,10 +66,14 @@ FortuneWindow::FortuneWindow(void)
 	
 	next->MakeFocus(true);
 	
-	// Calculate a good width for the window
-	SetSizeLimits(45 + close->Bounds().Width() + next->Bounds().Width(),30000,200,30000 ) ;
-	r = BScreen().Frame();
-	MoveTo((r.Width()-Bounds().Width())/2,r.Height() / 4);
+	BLayoutBuilder::Group<>(this, B_VERTICAL, B_USE_DEFAULT_SPACING)
+		.SetInsets(B_USE_WINDOW_INSETS)
+		.Add(sv)
+		.AddGroup(B_HORIZONTAL)
+			.Add(close)
```

Comment:
If you AddGlue() before and after the buttons, they are centered and keep together on resize.

Reply:
It really doesn't work well in my current situation.